src/monitoring_service/cli.py

```diff
@@ -53,6 +56,30 @@ def setup_logging(log_level: str) -> None:
     )
 
 
+def check_gas_reserve(web3: Web3, private_key: str) -> bool:
+    """ Check periodically for gas reserve in the account """
+    gas_price = web3.eth.gasPrice
+    gas_limit = GAS_REQUIRED_FOR_MS_MONITOR
+    estimated_required_balance = gas_limit * gas_price * 10
+    estimated_required_balance_eth = Web3.fromWei(estimated_required_balance, 'ether')
+    current_balance = web3.getBalance(private_key_to_address(private_key))
+    if current_balance < estimated_required_balance:
+        log.info('Missing gas reserve', required_wei=estimated_required_balance)
+        click.secho(
+            (
+                "WARNING\n"
+                "Your account's balance is below the estimated gas reserve of "
+                f"{estimated_required_balance_eth} Eth. You will be be unable"
+                "to perform on-chain transactions and cannot monitor any channels."
+                "Please add funds to your account as soon as possible."
+            ),
+            fg='red',
+        )
+        return False
+    else:
```

Comment:
nitpicky. I think in rest of codebase you guys avoid using else where not needed and I believe here it is not.

Reply:
Good point Thanks!

Reply:
@BOR4 always having your eyes open :)

Reply:
cheers guys!